easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
```

Comment:
OK, how to solve it? Should I rename version to `2016.01`? And every other "major" update will be increment of this number? We have to be consistent in the future.